packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -3,70 +3,124 @@ pragma solidity >=0.8.0 <0.9.0;
+        public __lenderMarketCommitments_deprecated;
+
+    // CommitmentId => commitment
+    mapping(uint256 => Commitment) public lenderMarketCommitments;
+
+    uint256 commitmentCount;
 
     /**
-     * @notice This event is emitted when a lender's commitment is submitted.
+     * @notice This event is emitted when a lender's commitment is created.
      * @param lender The address of the lender.
      * @param marketId The Id of the market the commitment applies to.
      * @param lendingToken The address of the asset being committed.
      * @param tokenAmount The amount of the asset being committed.
      */
-    event UpdatedCommitment(
-        address indexed lender,
-        uint256 indexed marketId,
-        address indexed lendingToken,
-        uint256 tokenAmount
+    event CreatedCommitment(
+        uint256 indexed commitmentId,
+        address lender,
+        uint256 marketId,
+        address lendingToken,
+        uint256 tokenAmount,
+
+        address collateralToken 
     );
 
     /**
-     * @notice This event is emitted when a lender's commitment has been removed.
+     * @notice This event is emitted when a lender's commitment is updated.
      * @param lender The address of the lender.
-     * @param marketId The Id of the market the commitment removal applies to.
-     * @param lendingToken The address of the asset the commitment removal applies to.
+     * @param marketId The Id of the market the commitment applies to.
+     * @param lendingToken The address of the asset being committed.
+     * @param tokenAmount The amount of the asset being committed.
      */
-    event DeletedCommitment(
-        address indexed lender,
-        uint256 indexed marketId,
-        address indexed lendingToken
+    event UpdatedCommitment(
+        uint256 indexed commitmentId,
+        address lender,
+        uint256 marketId,
+        address lendingToken,
+        uint256 tokenAmount,
+
+        address collateralToken 
     );
 
+    /**
+     * @notice This event is emitted when a lender's commitment has been deleted.
+     * @param commitmentId The id of the commitment that was deleted.
+     */
+    event DeletedCommitment(uint256 indexed commitmentId);
+
     /**
      * @notice This event is emitted when a lender's commitment is exercised for a loan.
-     * @param lender The address of the lender.
+     * @param borrower The address of the borrower.
      * @param marketId The Id of the market the commitment applies to.
      * @param lendingToken The address of the asset being committed.
      * @param tokenAmount The amount of the asset being committed.
      * @param bidId The bid id for the loan from TellerV2.
      */
     event ExercisedCommitment(
-        address indexed lender,
-        uint256 indexed marketId,
-        address indexed lendingToken,
+        uint256 indexed commitmentId,
+        address borrower,
+        uint256 marketId,
+        address lendingToken,
```

Comment:
these values are already being emitted in the update/create event and are repetitive here

Reply:
done

Reply:
the values have not been removed